src/tools/ScenarioMeasurement/SizeOnDisk/SizeOnDisk.cs

```diff
@@ -83,7 +87,7 @@ class SizeOnDisk
                 test.Name = scenarioName;
                 test.AddCounter(counters);
                 reporter.AddTest(test);
-                if (reporter.InLab && !String.IsNullOrEmpty(reportJsonPath))
+                if (reporter.InLab && !string.IsNullOrEmpty(reportJsonPath))
```

Comment:
It seems that previously we refer to ```String``` as a static class ... so I assume the lower case ```string``` means the same thing and it's just the convention's changed in ```net5.0```, or they are fundamentally different?

Reply:
It's not from .NET 5.0, it's a even older code-style suggestion. It exists since at least VS 2015. The suggestion is:

> IDE0049: Name can be simplified

- `string` is an alias of `String`, just like `int` is an alias for `Int32`.
- It is preferred to use the intrinsic predefined type keyword in member access expressions.
- `String` requires you to include `using System;`, while `string` does not.

This suggestion is enabled by default in VS 2017+ via:
    - Tools -> Options -> Text Editor -> C# -> Code Style -> Predefined type preferences -> For member access expressions -> Prefer predefined type